src/yb/tools/ts-cli.cc

```diff
@@ -94,6 +98,9 @@ DEFINE_bool(force, false, "If true, allows the set_flag command to set a flag "
             "which is not explicitly marked as runtime-settable. Such flag changes may be "
             "simply ignored on the server, or may cause the server to crash.");
 
+DEFINE_string(certs_dir_name, "",
```

Comment:
YB already have flags for certs dir. They are `certs_dir`, `certs_for_client_dir`, `certs_for_cdc_dir`. Why it is necessary to create new one?